### PR TITLE
[dist] Drop superflous common.sh from Vagrant

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

File: contrib/common.sh [hunk 1]
```diff
@@ -1,217 +0,0 @@
...
-  BASE_DIR=/vagrant/src/api/
-  # create log files to ensure they are owned by vagrant
-    for dir in log tmp
-    do
-      chown -R vagrant:users $BASE_DIR/$dir
-    done
-
-}
-
-function prepare_apache2 {
```
Comment (DavidKang) [inline contrib/common.sh]: Is this function not needed?

File: contrib/common.sh [hunk 2]
```diff
@@ -1,217 +0,0 @@
...
-}
-
-function configure_database() {
-  copy_example_file database.yml || return 1
-  check_for_databases && return 1
-  rake -f /vagrant/src/api/Rakefile db:create
-  rake -f /vagrant/src/api/Rakefile db:setup
-}
-
-function configure_search() {
```
Comment (DavidKang) [inline contrib/common.sh]: Is this function not needed?

Comment (hennevogel) [inline contrib/common.sh]: Exactly. foreman is calling ts:rebuild...

File: contrib/common.sh (hunk 1 above)
Comment (hennevogel) [inline contrib/common.sh]: Exactly. We don't use apache in the development environment

**Pull request merged**